### DISCUSSION TITLE
Create a button that mimics the shape and size of a mushroom button

### DISCUSSION THREAD

**Discussion by tonynogs** [Clooos/Bubble-Card#542] [Questions about config, custom styles and templates]
I've spent considerable time setting up a mobile dashboard that I like, but I want to make one of the buttons a pop up using bubble card. Is there a way to mimic the exact size and shape of this security button and turn it into a pop up?

![unnamed](https://github.com/Clooos/Bubble-Card/assets/162737403/5ae83cdb-5564-4c0e-84fb-08e08262c84d)

**Comment by MrBearPresident** [comment]
Did you solve this, because yes this is possible. 
Can this be closed?
